643_leetcode_problem.py

```diff
@@ -0,0 +1,15 @@
+class Solution:
+    def findMaxAverage(self, nums: List[int], k: int) -> float:
+    
+    
+        window = nums[:k]
+        size = len(nums)
+        maxSum = float('-inf')
```

Comment:
online 7, you could assign the first k(widow) average like maxSum = sum(window)/k